modules/BusinessActions/BusinessActions.php

```diff
@@ -299,8 +299,23 @@ public static function getAllByType($tabid, $type = false, $parameters = false,
 			}
 
 			if ($parameters) {
-				$link->linkurl = $strtemplate->merge($link->linkurl);
-				$link->linkicon= $strtemplate->merge($link->linkicon);
+				if (!empty($parameters['RECORD'])) {
+					$link->linkurl = $strtemplate->merge($link->linkurl);
+					$link->linkicon= $strtemplate->merge($link->linkicon);
```

Comment:
these two lines have to be outside this "if" because we will have situations where we have only the module to change